### PR TITLE
Clarify on how to import and use a Google font

### LINKED ISSUE #9729: "Import a Google Font" bug, and, apparently, this issue tracker's issue template markdown
#### Challenge Name

https://www.freecodecamp.com/challenges/import-a-google-font

#### Issue Description

The left sidebar text says "Now you can set Lobster as a font-family value on your h2 element", yet there is no h2 section in the `<style>` above, and there are no checkboxes in the bottom that explain that it needs to be added, or that it can be done by adding an inline `style="font-family: Lobster;"` to the h2 element. This is fairly elementary for experienced HTML users, but anyone actually new to coding would likely stop at this challenge for longer than intended. It's easy to tell that this isn't intended because in the very next challenge the steps are laid out for the user.

#### Browser Information

* Browser Name, Version: Chrome v51
* Operating System: Windows 7 64bit
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```
 <link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
<style>
  .red-text {
    color: red;
  }
  p {
    font-size: 16px;
    font-family: Monospace;
  }
</style>
<h2 class="red-text" style="font-family: Lobster;">CatPhotoApp</h2>
<p class="red-text">Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>
<p class="red-text">Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>
```

#### Screenshot

http://i.imgur.com/YyhcAjN.png

Wow. What the hell is wrong with the issue template markdown? Here's my source: http://i.imgur.com/5QuNoGI.png

### PR DESCRIPTION
**Pull request by erictleung**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #9729
#### Description

Remind user on how to create a CSS rule and that you'll need to create a new one for `h2` in order to complete the challenge.

### CODE REVIEW

**Comment by atjonathan** [comment]
### **LGTM :+1:**

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 1]
```diff
@@ -935,8 +935,9 @@
         "First, you'll need to make a <code>call</code> to Google to grab the <code>Lobster</code> font and load it into your HTML.",
         "Copy the following code snippet and paste it into the top of your code editor:",
         "<code>&#60;link href=\"https://fonts.googleapis.com/css?family=Lobster\" rel=\"stylesheet\" type=\"text/css\"&#62;</code>",
-        "Now you can set <code>Lobster</code> as a font-family value on your <code>h2</code> element.",
-        "Apply the <code>font-family</code> of <code>Lobster</code> to your <code>h2</code> element."
+        "Remember, before you can apply styles, like a new font, to an element, you'll need to create a CSS rule.",
+        "<blockquote>h2 {<br>  font-family: Sans-serif;<br>}</blockquote>",
```
Comment (atjonathan) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Should it not be lowercase, `sans-serif`?

Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: I don't think so but I could be wrong. I was copying the casing of the seed code. [This site](https://css-tricks.com/sans-serif/) has it as `Sans-Serif` but [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family) has it as `sans-serif`. I guess it's [case-insensitive](http://stackoverflow.com/a/26860699). 

Either way, my change is inconsistent. I'm fine by `sans-serif` or `Sans-Serif`.

**Comment by atjonathan** [comment]
Yeah we can just merge as is :smiley:
### LGTM :+1:

**Pull request merged**